lib/mux/video/assets.ex

```diff
@@ -121,6 +121,57 @@ defmodule Mux.Video.Assets do
 
   """
   def update_master_access(client, asset_id, params) do
-    Base.put(client, @path <> "/assets/" <> asset_id <> "/master-access", params)
+    Base.put(client, "#{@path}/#{asset_id}/master-access", params)
+  end
+
+  @doc """
+  Create a new playback ID.
+
+  Returns `{:ok, playback_id, %Telsa.Env{}}`.
+
+  ## Examples
+
+      iex> client = Mux.Base.new("my_token_id", "my_token_secret")
+      iex> {:ok, playback_id, _env} = Mux.Video.Assets.create_playback_id(client, "00ecNLnqiG8v00TLqqeZ00uCE5wCAaO3kKc", %{policy: "public"})
```

Comment:
I'm not sure why this one is failing, but the first thing that comes to mind is that maybe the mock is matching at the wrong place. Let's chat about this one Thursday, we can figure it out together.